contrib/scalar/scalar.c

```diff
@@ -156,23 +156,7 @@ static int set_recommended_config(int reconfigure)
 		{ "core.FSCache", "true", 1 },
 		{ "core.multiPackIndex", "true", 1 },
 		{ "core.preloadIndex", "true", 1 },
-#ifndef WIN32
 		{ "core.untrackedCache", "true", 1 },
```

Comment:
Since this sets `overwrite_on_reconfigure`, our users will upgrade successfully.